app/core/resources/constants/document_conversion.py

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: 2022 Zextras <https://www.zextras.com
+#
+# SPDX-License-Identifier: AGPL-3.0-only
+
+from app.core.resources.config_loader import read_config
+
+# LOG
+_document_conversion_section: str = "document_conversion"
+PROTOCOL: str = read_config(section=_document_conversion_section, value="protocol")
+IP: str = read_config(section=_document_conversion_section, value="ip")
+PORT: str = read_config(section=_document_conversion_section, value="port")
+SERVICE_ENDPOINT: str = read_config(
+    section=_document_conversion_section, value="service_endpoint"
+)
+CONVERT_API: str = read_config(
+    section=_document_conversion_section, value="convert_api"
+)
+
+BASE_ADDRESS: str = f"{PROTOCOL}://{IP}:{PORT}"
+FULL_SERVICE_ADDRESS: str = f"{BASE_ADDRESS}/{SERVICE_ENDPOINT}/"
```

Comment:
Maybe I'm wrong here but: is there a `/` missing before the `BASE_ADDRESS` or after the `PORT`?

Seeing the config.ini these are the values:
```
[document_conversion]
protocol = http
ip = 127.78.0.6
port = 20001
service_endpoint = services/docs/editor
convert_api = cool/convert-to
```

Reply:
BASE_ADDRESS = http://127.78.0.6:20001
FULL_SERVICE_ADDRESS = http://127.78.0.6:20001/services/docs/editor/

Where is the missing /?